test/fw/bin/pbs_benchpress

```diff
@@ -264,6 +271,8 @@ if __name__ == '__main__':
             testfiles = CliUtils.expand_abs_path(val)
         elif o == '-t':
             testsuites = val
+        elif o == '--extra-plugins':
+            extra_plugins = val
```

Comment:
Not something to do now, but we might consider using argparse instead of getopt.  It does a really good job with argument parsing.

Reply:
Absolutely agree. Also along with argparse we should really consider moving away from Nose as it is kind a dead project (last release was ~3.5 year back) and there is very good alternative called `pytest` (very actively maintained and much powerful/flexible than Nose)